python/hail/docs/scans.rst

```diff
@@ -46,33 +56,12 @@ The corresponding calls on MatrixTable fields would be:
     | 20:17640833   | ["A","C"]    |         3 |         224 |
     +---------------+--------------+-----------+-------------+
 
-    >>> ds_scan = ds.select_cols(ds.sample_qc.n_non_ref,
-    ...                          rolling_sum=hl.scan.sum(ds.sample_qc.n_non_ref))
-    >>> ds_scan.cols().show()
-    +----------------+-----------+-------------+
-    | s              | n_non_ref | rolling_sum |
-    +----------------+-----------+-------------+
-    | str            |     int64 |       int64 |
-    +----------------+-----------+-------------+
-    | C1046::HG02024 |         3 |           0 |
-    | C1046::HG02025 |         3 |           3 |
-    | C1046::HG02026 |         2 |           6 |
-    | C1047::HG00731 |         3 |           8 |
-    | C1047::HG00732 |         1 |          11 |
-    | C1047::HG00733 |         3 |          12 |
-    | C1048::HG02024 |         3 |          15 |
-    | C1048::HG02025 |         3 |          18 |
-    | C1048::HG02026 |         2 |          21 |
-    | C1049::HG00731 |         2 |          23 |
-    +----------------+-----------+-------------+
-
-
-Scans over entry fields are currently not supported.
+Scans over column fields can be done in a similar manner.
 
 .. DANGER::
 
     Computing the result of certain aggregators, such as ``hardy_weinberg``, can
```

Comment:
link to the aggregator. FYI, if https://github.com/hail-is/hail/pull/4032 goes in first, this will need to change to `hardy_weinberg_test`. Otherwise I'll rebase that